python/snewpy/models/ccsn.py

```diff
@@ -819,3 +461,5 @@ def get_fluence(self, t):
             fluence[k] = fl[:,idx]
 
         return fluence
+
+del Table, PinchedModel, RegistryModel, Parameter, deprecated, loaders
```

Comment:
In short: It's not needed now.

This was done when I tried to make sphinx documentation to be generated from this module.
It didn't see the models unless I set `:members-inherited:`, which included all kinds of stuff we imported from other modules (namely `Table, PinchedModel, RegistryModel, Parameter, deprecated, loaders`), so I had to delete them.

Then I realized that it was because the model classes were created by the `RegistryModel` decorator in the `models.registry_model` module, so I copied their `__module__` attribute from the decorated base class (defined in `models.ccsn`), and everything works now.

On the other hand I like this `del` statement, it limits what a user can import from this module directly. I've seen some packages use this quite a lot, to make interface minimal. But maybe there is a better way to achieve this.

Reply:
For the consistency I remove it